help/en/html/apps/Errors.shtml

```diff
@@ -0,0 +1,216 @@
+<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01//EN"
+"http://www.w3.org/TR/html4/strict.dtd">
+
+<html lang="en">
+<head>
+  <meta name="generator" content=
+  "HTML Tidy for Mac OS X (vers 31 October 2006 - Apple Inc. build 15.17), see www.w3.org">
+  <!-- Copyright Bob Jacobsen 2008 - 2016 -->
+
+  <title>JMRI Setup - Error Messages and Codes</title>
+
+  <!--#include virtual="/Style.shtml" -->
+
+</head>
+
+<body>
+  <!--#include virtual="/Header.shtml" -->
+  <div id="mBody">
+    <!--#include virtual="Sidebar.shtml" -->
+    <div id="mainContent">
+      <!-- Page Body -->
+
+      <h1>JMRI Setup: Error Messages and Codes</h1>
+      <!-- THIS WILL BE THE MASTER COPY: In process of removing copies from help/en/manual/DecoderPro3/Error.shtml and help/en/html/tools/programmer/ 
+      and help/en/html/setup/Errors.shtml   @jerryg2003 08/07/2020 -->
+
+      <p>Various things can go wrong when JMRI tries to work with
+      your DCC command station to program a DCC decoder. This page
+      lists the error message that can result, and describes what
+      they mean.</p>
+
+      <h2>JMRI Error Codes</h2>
+
+      <dl>
+        <dt><strong>301 - no locomotive detected</strong></dt>
+
+        <dd>The command station has reported that it doesn't see a
+        locomotive on the programming track.</dd>
+
+        <dd>
```

Comment:
I think these should be formatted as just a single `<dd></dd>`, with multiple `<p>` paragraphs in between as needed.  You've got more than one `<dd></dd>` associated with the `<dt></dt>` blocks, which might not format properly.

Reply:
Bob - 

This file is almost an exact copy of the existing file setup/Errors.shtml so you can see how it looks (I didn't change any formatting).  Also an exact copy of manual/DecoderPro/Error.shtml, DecoderPro3/Error.shtml, and toos/programmer/messages.shtml.  [It is my intention to delete all of these others after I fix all links.]  However, my point is that I didn't change existing formatting and it looks OK when I display it in Chrome on my Windows 10 machine. 

Jerry

Reply:
A `<dl></dl>` is a "dictionary list".  It's [defined as](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-dl-element.html) a series of `<dt>/<dt>` "terms" and `<dd></dd>` "definitions".

Its [fine to have multiple definitions](https://www.w3.org/TR/2011/WD-html5-author-20110809/the-dl-element.html) for a single term, but then they're considered 'alternative' definitions:  'multiple paragraphs forming part of the same value must all be given within the same dd element.'

I agree that most browsers don't make any distinction between multiple paragraphs within a value and multiple values.  Safari (my default browser) makes a visual distinction when doing default formatting, but doesn't make one with JMRI's CSS. John Plocher (@plocher)might have set that up intentionally.

There are places where it matters, though. We've recently seen one surface: a blind new user has been trying to work with JMRI via a screen reader.  He's uncovered several things in the program itself, most of which are now fixed.  But he's also sensitized me a bit to how we use HTML.  The screen reader is _really_ good at working with standard HTML, but it's also quite literal:  It represents the _mark up_ aspect of HT**ML** quite specifically.

So I agree that it's not a reason to hold this PR. I'll merge it momentarily.  But it might be good to add to your every-growing list of things that could be improved in the documentation.

Thanks for your patience with this.